extensions/eclipse-che-theia-filesystem/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@eclipse-che/theia-filesystem-extension",
+  "keywords": [
+    "theia-extension"
+  ],
+  "version": "0.0.1",
+  "files": [
+    "lib",
+    "src"
+  ],
+  "dependencies": {
+    "@theia/filesystem": "1.4.0-next.89694867"
```

Comment:
Why the specific version?
Normally, we depend on Theia `next` everywhere.
The specific version is already pinned in [`yarn.lock`](https://github.com/eclipse/che-theia/blob/503ca56a3c1a7e7c9ccab1f975b49e8ecd0ea993/yarn.lock#L2255-L2257).

Reply:
With the next tag there're new changes for the file system api may income. This value can be modified to `next` when che-theia will be aligned to theia's master head. Or is it okay, that we will use `next` version for the filesystem library?

Reply:
I see. I missed that moment.